File: Leetcode - Python/1523_math_countOdds.py
# https://leetcode.com/problems/count-odd-numbers-in-an-interval-range/

class Solution(object):
    def countOdds(self, low, high):
        res = (high - low) / 2; 
        return res + 1 if (low % 2 != 0 or high % 2 != 0) else res; 
        

File: Leetcode - Java/1523_math_countOdds.java
// https://leetcode.com/problems/count-odd-numbers-in-an-interval-range/

class Solution {
    public int countOdds(int low, int high) {
        int res = (high - low) / 2;
        return (low % 2 != 0 || high % 2 != 0) ? res + 1 : res; 
    }
}